_source/_layouts/author.html

```diff
@@ -35,6 +35,9 @@ <h1 class="BlogPost-title">
 		    </div>
 		{% endif %}
 
+    {% assign first_name = page.full_name | split: " "[0]%}
```

Comment:
Does this work on someone that has three words in their name?

Reply:
It looks like that `first_name` isn't even used later